.env.sample

```diff
@@ -0,0 +1 @@
+VUE_APP_API_KEY=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
```

Comment:
Let's replace this `api_key` value by `some_api_key_value`

Reply:
I agree. Maybe `VUE_APP_SENDGRID_API_KEY`.

Reply:
Done!

Reply:
@ArmandoGraterol please rename the env by the name above